src/types.ts

```diff
@@ -0,0 +1,24 @@
+export const WEEK_DAYS = [
+  'monday',
+  'tuesday',
+  'wednesday',
+  'thursday',
+  'friday',
+  'saturday',
+  'sunday',
+] as const
+
+export type weekDay = (typeof WEEK_DAYS)[number]
```

Comment:
Used in couple places so it was beneficial to extract `(typeof WEEK_DAYS)[number]` to type